source/clear-linux/get-started/virtual-machine-install/gce.rst

```diff
@@ -135,67 +172,88 @@ instance of |CL-ATTR| on `Google Cloud Platform`_ (GCP).
+     `Best practices for Compute Engine regions selection`_.
+
+   * Under *Boot disk*, click the ``Change`` button.
 
      .. figure:: figures/gce/30-create-vm.png
         :scale: 50 %
         :alt: Use custom image while creating Clear Linux VM instance
 
+        Figure 15: Use custom image
+
    * Select the *Custom images* tab for using Clear Linux OS GCE image.
 
      .. figure:: figures/gce/31-select-boot-disk.png
         :scale: 50 %
         :alt: Select Clear Linux boot disk to create a VM instance
 
+        Figure 16: Select Clear Linux boot disk to create a VM instance
+
    * Scroll down to the bottom of the VM instance creation page,
      expand the *Management, security, disks, networking, sole tenancy* group.
 
      .. figure:: figures/gce/40-clear-vm-security.png
         :scale: 50 %
         :alt: Clear Linux requires setting up SSH keys
 
+        Figure 17: Set up SSH keys
+
      .. note::
         |CL| does not allow SSH login with a root account by default.
-        As a result, you need to configure the VM instance with your
-        SSH public key, so that you are able to remote access to it.
+        As a result, you must configure the VM instance with your
+        SSH public key, so that you are able to access it remotely.
 
-        Refer also to :ref:`security` for more details.
+        Refer to :ref:`security` for more details.
 
    * Click the *Security* tab, copy and paste your SSH public key:
 
      .. figure:: figures/gce/40-ssh-key.png
         :scale: 50 %
         :alt: Set SSH key for remote login
 
+        Figure 18: Set SSH key for remote login
+
      .. warning::
 
-        The username is assigned from characters preceding ``@`` in the
-        email address, included in the SSH key. The dot symbol "." is not
-        allowed, as it is an invalid character while creating user accounts
-        in |CL|.
+        The username is assigned from characters preceding ``@`` in the email
+        address, included in the SSH key. The dot symbol "." is not allowed,
+        because it is an invalid character while creating user accounts in
+        |CL|.
 
    * Click the ``Create`` button to create the |CL| VM.
 
-#. The Clear Linux VM instance is created and offered a public IP address:
+#. The Clear Linux VM instance is created and assigned a public IP address:
 
    .. figure:: figures/gce/41-vm-created.png
       :scale: 50 %
       :alt: Clear Linux VM instance is created and started
 
-#. You can now SSH login to the VM by using the IP address obtained in the
+      Figure 19: Clear Linux VM instance is created and started
+
+#. You can now SSH login to the VM using the IP address obtained in the
    previous step, and the username associated with the SSH public key:
 
    .. figure:: figures/gce/42-ssh-vm.png
       :scale: 50 %
-      :alt: SSH log in to the Clear Linux VM
+      :alt: SSH login to the Clear Linux VM
+
+      Figure 20: SSH login to Clear Linux VM
+
+Related topics
```

Comment:
@ttzeng  Any other related topics that I should link here?

Reply:
@MCamp859, maybe we can put **similar** topics here, though they are not related to Google cloud... they are tutorials running CL on [Microsoft Azure](https://clearlinux.org/documentation/clear-linux/tutorials/azure) and [Amazon AWS](https://clearlinux.org/documentation/clear-linux/tutorials/aws-web) clouds respectively.